modules/afsmtp/afsmtp.c

```diff
@@ -57,11 +56,9 @@ typedef struct
   gchar *host;
   gint port;
 
-  gchar *subject;
   AFSMTPRecipient *mail_from;
   GList *rcpt_tos;
   GList *headers;
-  gchar *body;
 
   LogTemplate *subject_tmpl;
   LogTemplate *body_tmpl;
```

Comment:
Please rename `subject_tmpl` and `body_tmpl` to `subject_template` and `body_template`. It would make the code consistent (if you have already renamed the fields in `AFSMTPRecipient` and `AFSMTPHeader` )